docs/src/numberfield.md

```diff
@@ -153,18 +169,27 @@ degree(::AbsSimpleNumField)
 
 **Examples**
 
-```julia
-R, x = polynomial_ring(QQ, "x")
-K, a = number_field(x^3 + 3x + 1, "a")
+```jldoctest
+julia> R, x = polynomial_ring(QQ, "x")
+(Univariate polynomial ring in x over QQ, x)
+
+julia> K, a = number_field(x^3 + 3x + 1, "a")
+(Number field of degree 3 over QQ, a)
+
+julia> d = a^2 + 2a - 7
+a^2 + 2*a - 7
+
+julia> m = gen(K)
+a
+
+julia> c = coeff(d, 1)
+2
 
-d = a^2 + 2a - 7
-m = gen(K)
+julia> is_gen(m)
+true
 
-c = coeff(d, 1)
-is_gen(m)
-q = degree(K)
-r, s = signature(K)
```

Comment:
Has been removed in https://github.com/Nemocas/Nemo.jl/pull/778, but they forgot the example